frontend/globals/unity-util.ts

```diff
@@ -313,7 +313,12 @@ export class UnityUtil {
 
 	public static viewpointReturned(vpInfo) {
 		if (UnityUtil.vpPromise != null) {
-			const viewpoint = JSON.parse(vpInfo);
+			let viewpoint = {} ;
+			try {
+				viewpoint = JSON.parse(vpInfo);
+			} catch {
+				console.error("Failed to parse viewpoint", vpInfo);
```

Comment:
Not sure of the surrounding context but would it make sense to reject the promise so it can be handled upstream?

Reply:
That depends if the user would rather to save a issue without a viewpoint or get a warning instead. I kind of assume the former.

Reply:
If we can't get a viewpoint, can it be made to look at the centre of the model?

Reply:
It should already do that, I had to implement this for Issues made from Node Red